ccdproc/tests/test_image_collection.py

```diff
@@ -148,6 +148,19 @@ def test_hdus_masking(self, triage_setup):
         new_data = np.array(collection.summary_info)
         assert (new_data == old_data).all()
 
+    def test_multiple_extensions(self, triage_setup):
+        ext1 = fits.PrimaryHDU()
+        ext2 = fits.ImageHDU(name = 'MASK')
+        hdulist = fits.hdu.hdulist.HDUList([ext1, ext2])
+
+        hdulist.writeto(os.path.join(triage_setup.test_dir, 'multi-extension.fits'))
+        ic = image_collection.ImageFileCollection(triage_setup.test_dir, keywords='*',
+                                    filenames = ['multi-extension.fits'], extension='MASK')
+        list1 = [key.lower() for key in ext2.header]
+        list2 = ic.summary_info.colnames[1:]
+
+        assert list1 == list2
```

Comment:
Maybe this could also check that the `ImageFileCollection` of `ext=0` is different from the one in `ext='MASK'`.